### ISSUE TITLE
Where we configure the openid provider?

### ISSUE THREAD

**Issue by ramiroluz** [collective/pas.plugins.oidc#52]
The README says to configure the openid profile but I could not find any instruction in neither plugins (pas.plugins.oidc and volto-authomatic)  to learn where to configure the OpenID provider.
 
![image](https://github.com/collective/pas.plugins.oidc/assets/293174/6448c998-cafe-40e5-b7c0-619be6d69105)

**Comment by ramiroluz** [comment]
I am not sure, but I think we need 3 plugins to use volto and pas.plugins.oidc.

1. pas.plugins.oidc
2. volto-authomatic
3. pas.plugins.authomatic

Then we configure the oidc provider in pas.plugins.authomatic (trying this).

**Comment by erral** [comment]
Nope, pas.plugins.oidc is enough. You need to install it and then go to the ZMI, you will find the oidc plugin inside acl_users.

Then you need volto-authomatic for Volto integration .

**Comment by ramiroluz** [comment]
We found the problem, I was using an old version of oidc. It need to be > 2
